test/JDBC/expected/money_aggregate-vu-prepare.out

```diff
@@ -0,0 +1,188 @@
+CREATE TABLE TestMoneyTable (
+    ID INT IDENTITY(1,1) PRIMARY KEY,
+    Amount MONEY,
+    Description NVARCHAR(100)
+);
+GO
+
+-- Insert test data
+INSERT INTO TestMoneyTable (Amount, Description) VALUES 
+(100.50, 'Item 1'),
+(200.75, 'Item 2'),
+(150.25, 'Item 3'),
+(300.00, 'Item 4'),
+(250.50, 'Item 5'),
+(NULL, 'Null Amount');
+GO
+~~ROW COUNT: 6~~
+
+
+CREATE PROCEDURE get_column_info_p1
+    @table_name text
+AS
+BEGIN
+    SELECT c.[name] AS column_name,
+           t.[name] AS [type_name],
+           c.[max_length],
+           c.[precision],
+           c.[scale]
+    FROM sys.columns c
+        INNER JOIN sys.types t
+            ON c.user_type_id = t.user_type_id
+    WHERE object_id = object_id(@table_name)
+    ORDER BY c.[name];
+END
+GO
+
+-- Test Case 1: MAX aggregation
+SELECT MAX(Amount) AS MaxAmount
```

Comment:
lets also make sure that we create other type of dependent objects like view/function/procedure and check metadata to ensure that its correct.